websites/R/RTLplay/presence.ts

```diff
@@ -0,0 +1,555 @@
+				presenceData.state = episodeName !== "" ? episodeName : mediaName;
+
+				if (poster) {
+					presenceData.largeImageKey = await getShortURL(
+						document
+							.querySelector("#content > script")
+							.textContent.match(
+								/window\.App\.playerData\s*=\s*\{[\s\S]*?poster:\s*"(.*?)",/
+							)[1]
+							.replace(/\\u0026/g, "&")
+							.replace(/\\/g, "")
+					);
+				}
+
+				if (seasonNumber && episodeNumber) {
+					presenceData.largeImageText = `${
+						(await strings).season
+					} ${seasonNumber} - ${(await strings).episode} ${episodeNumber}`;
+				}
+
+				if (time) {
+					presenceData.endTimestamp = presence.getTimestamps(
+						presence.timestampFromFormat(
+							document
+								.querySelector("span.playerui__controls__stat__time")
+								.textContent.split("/")[0]
+								.trim()
+						),
+						presence.timestampFromFormat(
+							document
+								.querySelector("span.playerui__controls__stat__time")
+								.textContent.split("/")[1]
+								.trim()
+						)
+					)[1];
+				}
+
+				if (buttons) {
+					presenceData.buttons = [
+						{
+							label:
+								episodeName !== ""
+									? strings.buttonWatchEpisode
+									: (await strings).buttonWatchMovie, // Need to be a general string
+							url: href, // We are not redirecting directly to the raw video stream, it's only the media page
+						},
+					];
+				}
+			}
+
+			if (exist("i.playerui__icon--name-play")) {
+				// State paused
+				presenceData.smallImageKey = Assets.Pause;
+				presenceData.smallImageText = (await strings).pause;
+			} else {
+				presenceData.smallImageKey = Assets.Play;
+				presenceData.smallImageText = (await strings).play;
+			}
+			break;
+		}
+		/* MEDIA PAGE (Page de media)
+
+		(https://www.rtlplay.be/rtlplay/salvation~2ab30366-51fe-4b29-a720-5e41c9bd6991) */
+		case pathname.split("/")[2].length > 15: {
+			presenceData.smallImageKey = Assets.Viewing;
+			presenceData.smallImageText = (await strings).viewAPage;
+
+			if (privacy) presenceData.details = (await strings).viewAPage;
+			else {
+				let subtitle = document.querySelector(
+					'dd.detail__meta-label[title="Année de production"]'
+				)
+					? document.querySelector(
+							'dd.detail__meta-label[title="Année de production"]'
+					  ).textContent
+					: ""; // Get Release Year
+				subtitle += document.querySelector(
+					'dd.detail__meta-label[title="Durée"]'
+				)
+					? ` - ${
+							document.querySelector('dd.detail__meta-label[title="Durée"]')
+								.textContent
+					  }`
+					: ""; // Get Duration
+				for (
+					let i = 0;
+					document.querySelectorAll("dl:nth-child(1) > dd > a").length > i;
+					i++ // Get Genres
+				) {
+					subtitle += ` - ${
+						document.querySelectorAll("dl:nth-child(1) > dd > a")[i].textContent
+					}`;
+				}
+
+				presenceData.details = (await strings).viewPage;
+				presenceData.state =
+					document.querySelectorAll("h1.detail__title")[0].textContent;
```

Comment:
If you're getting the first element user querySelector() instead of querySelectorAll()
```suggestion
					document.querySelector("h1.detail__title").textContent;
```